reference/uitpas.json

```diff
@@ -2577,30 +2577,7 @@
                 "schema": {
                   "$ref": "#/components/schemas/Pass"
                 },
-                "examples": {
-                  "Example with warning message": {
-                    "value": {
-                      "passholderId": "b34c19fc-bcf9-4d10-aeb2-6bc9efd38b68",
-                      "uitpasNumber": "0560002524314",
-                      "points": 12,
-                      "postalCode": "1000",
-                      "messages": [
-                        {
-                          "level": "WARNING",
-                          "text": "Je sociaal tarief is verlopen. Contacteer je UiTPAS balie."
-                        }
-                      ]
-                    }
-                  },
-                  "Example without messages": {
-                    "value": {
-                      "passholderId": "b34c19fc-bcf9-4d10-aeb2-6bc9efd38b68",
-                      "uitpasNumber": "0560002524314",
-                      "points": 12,
-                      "postalCode": "1000"
-                    }
-                  }
-                }
+                "examples": {}
```

Comment:
I think you need to remove the `examples` property completely for Stoplight to "inherit" the examples from the schema in the docs.